common/changes/@microsoft/rush/chao-enable-pnpm-sync_2024-01-22-21-34.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Integrate pnpmSyncPrepare whenever rush calls `pnpm install` ",
```

```suggestion
      "comment": "Add `enablePnpmSyncForInjectedDependenciesMeta` to experiments.json; it is part of an upcoming feature for managing PNPM \"injected\" dependencies: https://www.npmjs.com/package/pnpm-sync",
```

Change logs should describe user impact rather than implementation details; see [these guidelines](https://rushjs.io/pages/best_practices/change_logs/)

---

Also we need to add a **README.md** for your package; I can help with this